src/components/ui/TopographyPattern.tsx

```
@@ -0,0 +1,37 @@
+  color = 'currentColor',
+  opacity = 0.1,
+}: TopographyPatternProps) {
+  return (
+    <svg
+      className={className}
+      width="100%"
+      height="100%"
+      viewBox="0 0 1000 1000"
+      xmlns="http://www.w3.org/2000/svg"
+      preserveAspectRatio="none slice"
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)